validator/example.config.toml

```diff
@@ -0,0 +1,6 @@
+rpc_url = "http://127.0.0.1:8899"
+ws_url = "ws://127.0.0.1:8900"
+program_id = "9fd7GDygnAmHhXDVWgzsfR6kSRvwkxVnsY8SaSpSH4SX"
+genesis_hash = "AXO4arKprlSJUQssh8aJxLIWFX5sObiG2Nd2817cfvY="
+keypair = [48,123,8,80,248,0,217,142,124,193,95,24,168,139,214,136,147,210,168,135,26,36,162,89,150,185,99,191,247,135,78,111,12,8,4,81,129,165,153,230,192,225,51,119,216,14,69,225,73,7,204,144,39,213,91,255,136,38,95,131,197,4,101,186]
```

Comment:
@dhruvja this key is rubbish, right?

Reply:
i just picked it from some random github repo

Reply:
lol